vendor/github.com/IBM/ibm-csi-common/pkg/messages/messages.go

```diff
@@ -42,9 +44,22 @@ func (msg Message) Error() string {
 
 // Info ...
 func (msg Message) Info() string {
+	/*If the BackendError is from library e.g BackendError: {Trace Code:920df6e8-6be9-4b4a-89e4-837ecb3f513d,
+	Code:InvalidArgument ,Description:Please check parameters,RC:400 Bad Request}
+	*/
 	if msg.BackendError != "" {
-		return fmt.Sprintf("{RequestID: %s, Code: %s, Description: %s, BackendError: %s, Action: %s}", msg.RequestID, msg.Code, msg.Description, msg.BackendError, msg.Action)
+		return fmt.Sprintf("{RequestID: %s, BackendError: %s, Action: %s}", msg.RequestID, msg.BackendError, msg.Action)
 	}
+	/*If there is CSIError from Driver side e.g
+	Error: XYZ is mandatory, Action: Please provide valid parameters
+	*/
```

Comment:
@MeetanshiGupta can we build custom image and reproduce the error messages and see how it looks. I think the BackendError: error will come as Error:. Ideally it should not break anything but the format will be reduced and BackendError key word wont come but instead Error until we do below changes.

@arahamad If we dont want this then we need to similar changes like file driver
**Library:** https://github.com/IBM/ibmcloud-volume-file-vpc/pull/44/files
**Driver:** https://github.com/IBM/ibm-vpc-file-csi-driver/pull/103/files